README.md

```diff
@@ -347,6 +347,39 @@ You can customise the Chatwoot placeholder by passing the following props to the
 
 </details>
 
+<details>
+<summary id="hubspot">Hubspot</summary>
+
+To use Hubspot import the `LiveChatLoaderProvider` and set the `provider` prop
+as `hubSpot` and the `providerKey` prop as your Hubspot API Key.
+
+Then import the `Hubspot` component.
+
+```jsx
+import { LiveChatLoaderProvider, Hubspot } from 'react-live-chat-loader'
+
+export default class App extends React.Component {
+  render() {
+    return (
+      <LiveChatLoaderProvider providerKey="asdjkasl123123" provider="hubspot">
+        /* ... */
+        <Hubspot />
+      </LiveChatLoaderProvider>
+    )
+  }
+}
+```
+
+You can customise the Hubspot placeholder by passing the following props to the
+`Hubspot` component:
+
+- `backgroundColor`: The background color of the placeholder
+- `loader`: A react component shown while the Hubspot libraries are loading
+
+</details>
+
+<details>
```

Comment:
```suggestion
```

This causes a regression that hides the **Adding a Provider** content in a collapsed `<details>` tag.

Reply:
Thanks for the catch! I've just committed your suggestion 🙌